apps/ios/SimpleX SE/ShareError.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  ShareErr.swift
+//  SimpleX SE
+//
+//  Created by Levitating Pineapple on 11/07/2024.
+//  Copyright © 2024 SimpleX Chat. All rights reserved.
+//
+
+import Foundation
+import SimpleXChat
+
+enum ShareError: LocalizedError {
```

Comment:
don't think we need that, assuming we want to show some alert.

We used that pattern a lot, out of ignorance I think, but you can just use ChatResponse and Alert, not sure there is a need to have some enum in the middle